src/test/java/io/stargate/sgv2/jsonapi/api/v1/tables/FindOneTableIntegrationTest.java

```diff
@@ -31,6 +33,16 @@ public final void createDefaultTables() {
             "name",
             Map.of("type", "text")),
         "id");
+    createTableWithColumns(
```

Comment:
we should prob have a discussion at some point about what the unit tests should cover and what the integrations should, if we decide they should be independant and can cover the same ground that is cool.

Reply:
Right, I think some overlap can be useful, but doubling of all coverage is wasteful. Right now I am interested in end-to-end translation of exceptions,  which is partly why I add overlapping case.

Not sure how to guarantee good balance; I think we have leaned bit too much  on integration tests earlier, partly as operation unit test have been bit too mock-based (and IMO not necessarily giving confidence that Operations actually work the same way end-to-end).